backend/selectgen.ml

```diff
@@ -642,6 +644,12 @@ method select_operation op args _dbg =
   | (Cintoffloat, _) -> (Iintoffloat, args)
   | (Cvalueofint, _) -> (Ivalueofint, args)
   | (Cintofvalue, _) -> (Iintofvalue, args)
+  | (Catomic {op; size}, [src; dst]) ->
```

Comment:
Something like this that explicitly lists the cases would be safer:
```suggestion
  | (Catomic {op=(Fetch_add|Fetch_sub) as op; size}, [src; dst]) ->
```

Reply:
Listed